src/templates/csharp/TestApp.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Remote;
+
+using NUnit.Framework;
+using System.Threading;
+using System.Collections.Generic;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Android;
+using OpenQA.Selenium.Appium.iOS;
+using OpenQA.Selenium.Support.UI;
+using System.Drawing;
+
+namespace AppiumTest
+{
+    public class Tests : TestBase
+    {
+        [SetUp]
+        public void BeforeAll() {
+            Assert.AreNotEqual(
+                "Please update value for the KobitonApiKey constant first. See more at README.md file.",
+                "your_kobiton_api_key",
+                Config.KobitonApiKey
+            );
+        }
+
+        {{testCases}}
```

Comment:
The generated project has empty test case body. It should call FindOnlineDevice, Setup, RunTest
<img width="571" alt="image" src="https://github.com/kobiton/appium-script-generator/assets/15189024/1dff745e-9a7d-46ec-8640-ff3c9144d943">